packaging/nix/default.nix

```diff
@@ -0,0 +1,42 @@
+{ pkgs ? import <nixpkgs> {} }:
+# Based on nixpkgs/pkgs/applications/graphics/darktable/default.nix
+
+let stdenv = pkgs.stdenv;
+in stdenv.mkDerivation rec {
+  name = "darktable-git";
+
+  src = pkgs.lib.cleanSource ./.;
+
+  buildInputs = with pkgs; with xorg; with gnome2;
```

Comment:
are you sure about these deps? e.g. libgnome_keyring is dead. libglade, SDL are not deps.
and libpthreadstubs looks strange, is it not handled by std library?